### PR TITLE
/cancel powerup

### PR DESCRIPTION
**Pull request by TendTo**
- The /cancel command now also allows the user to remove a post that is still pending, allowing him to spot immediately.
- The user tags used to show who, among the admins, approved or rejected a pending post, can be turned in inoffensive text (removing the @ at the beginning).
- Fixed a pesky bug that made the bot unresponsive to further /spots and forced the user to use the /cancel command.
- Some very minor refactoring.

### CODE REVIEW

File: modules/utils/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
 """Modules that provide various util"""
+
+REACTION = {'0': "👎", '1': "👍", '2': "🤣", '3': "😡", '4': "🥰"}
```
Comment (Helias) [inline modules/utils/__init__.py:3]: change '0', '1', with 'thumbsdown', 'thumbsup' etc.

File: modules/data/meme_data.py [hunk 2]
```diff
@@ -160,6 +160,28 @@ def remove_pending_meme(g_message_id: int, group_id: int):
...
+        Returns:
+            tuple: if a pending post was deleted, returns its id and the admin group id, return None otherwise
+        """
+        result_row = DbManager.select_from(select='g_message_id, group_id',
+                                           table_name='pending_meme',
+                                           where='user_id = %s',
+                                           where_args=(user_id, ))
+
+        if len(result_row) == 0:  # there was no pending meme
+            return None, None
```
Comment (Helias) [inline modules/data/meme_data.py:179]: :heart:

**Review by Helias — CHANGES_REQUESTED** [review]

**Review by drendog — APPROVED** [review]

**Review by drendog — COMMENTED** [review]
****

**Comment by Helias** [comment]
😢

**Pull request merged**